RadixWallet/Clients/EntitiesVisibilityClient/EntitiesVisibilityClient+Interface.swift

```diff
@@ -13,8 +13,34 @@ extension EntitiesVisibilityClient {
 		public let hiddenPersonasCount: Int
 	}
 
-	public typealias HideAccount = @Sendable (Profile.Network.Account) async throws -> Void
-	public typealias HidePersona = @Sendable (Profile.Network.Persona) async throws -> Void
+	public typealias HideAccounts = @Sendable (Set<Profile.Network.Account.ID>) async throws -> Void
```

Comment:
also switched to `ID` based, since it is what is needed (used), always good with *inclusive* arguments, and *specific* return values.